packages/common-ui-web/src/components/CreateInspection/CreateInspection.tsx

```diff
@@ -53,6 +55,7 @@ export const CreateInspection = i18nWrap(({ lang, onInspectionCreated }: CreateI
         .then((res) => {
           loading.onSuccess();
           setInspectionId(res.id);
+          setUserId(res.id);
```

Comment:
Is it normal to use the inspection ID for the user ID?

Reply:
Yes, it allows us to easily tracks the status of a inspection. Actually, I can add an other tag with the client infos(mail...)